package.json

```
@@ -26,49 +26,66 @@
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.23.0",
-    "ember-cli-htmlbars": "^5.3.1"
+    "ember-auto-import": "^2.4.0",
+    "@glimmer/component": "^1.0.4",
+    "@glimmer/tracking": "^1.0.4",
+    "ember-cli-babel": "^7.26.10",
+    "ember-cli-htmlbars": "^5.7.2",
+    "uuid": "^8.3.2"
   },
   "peerDependencies": {
-    "@lblod/marawa": "^0.8.0-beta.2",
-    "ember-concurrency": "^1.0.0"
+    "@appuniversum/ember-appuniversum": "^1.0.2",
+    "@lblod/ember-rdfa-editor": "^0.50.0",
```

I think it's best to have to peer dependency quite broad
```suggestion
    "@lblod/ember-rdfa-editor": ">= 0.50.0",
```